node-authorizer/pkg/client/client.go

```diff
@@ -43,6 +43,14 @@ func New(config *Config) error {
 		zap.String("kubeconfig", config.KubeConfigPath),
 		zap.String("registration-url", config.NodeURL))
 
+	// @step: if we have a kubecfg already we can skip it
+	if utils.FileExists(config.KubeConfigPath) {
+		utils.Logger.Info("skipping the client authorization as kubecfg found",
+			zap.String("kubecfg", config.KubeConfigPath))
```

Comment:
kubecfg or kubeconfig, bit confused with a few lines up in the file?

Reply:
I never really got the standard but as far as I can remember Kubernetes uses the KUBECONFIG environment variable, thus I’d keep it consistent - especially in output.

Reply:
agreed and fixed in https://github.com/kubernetes/kops/pull/5868/commits/8746f9327fa7576b3b03b187a02df60feb73a4a7